main.py

```diff
@@ -29,8 +40,7 @@ def __init__(self):
         self.processor = WhisperProcessor.from_pretrained("openai/whisper-base")
 
     def forward(self, data):
-        res = self.processor.batch_decode(data, skip_special_tokens=True)
-        return [{"text": msg} for msg in res]
+        return self.processor.batch_decode(data, skip_special_tokens=True)
```

Comment:
The returned data is still a JSON string. Need to override the serialize method.